ldk/go/keyboardService_test.go

```diff
@@ -50,7 +51,7 @@ func TestKeyModifiers(t *testing.T) {
 				for j, fn := range funcs {
 					fnName := runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
 					if i == j {
-						t.Logf("testing %d -> %s", bit, fnName)
+						// t.Logf("testing %d -> %s", bit, fnName)
```

Comment:
Is the log needed?

Reply:
Honestly I think I'd rather keep it commented out; it makes the test output really noisy.